### ISSUE TITLE
should change URL

### ISSUE THREAD

**Issue by fjwldap** [atom/atom#16277]
### Prerequisites

* [ ] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: http://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

[Description of the issue]

In file atom/apm/package.json line 6, there are URL, "https://github.com/atom/atom.git".
But if we go to that site, the  URL changes to "https://github.com/atom/atom". Maybe the url has changed.
So we should change URL to it.

### Steps to Reproduce

1. [First Step]
2. [Second Step]
3. [and so on...]

**Expected behavior:** [What you expect to happen]

**Actual behavior:** [What actually happens]
URL changes to "https://github.com/atom/atom
**Reproduces how often:** [What percentage of the time does it reproduce?]
now, every time.

**Comment by mpboom** [comment]
The actual URL for **getting a Git repository** from GitHub is `https://github.com/user/repo.git`.  The URL where the repository is on the **GitHub website** is `https://github.com/user/repo`.

However, you are able to clone a repository with the URL without the `.git` and if you visit the URL with the `.git` in the browser it redirects to the one without it - you can use the `.git` **and** the URL without `.git` for both visiting a project on the GitHub website **and** retrieve a repository.

Bottom line - you can use both URL's. In the `package.json` file you should officially specify the URL with `.git` - that is because that URL should point directly towards a repository. When you visit it with a web browser, though, GitHub redirects you towards the project page (the URL without `.git`) - but that is just a convenience feature.

**Comment by 50Wliu** [comment]
Thank you @mpboom for the detailed explanation :sparkles:.
